corehq/apps/accounting/models.py

```diff
@@ -1730,6 +1733,9 @@ def get_active_subscription_by_domain(cls, domain_name_or_obj):
     @classmethod
     @quickcache(['domain_name'], timeout=60 * 60)
```

Comment:
how about adding this to avoid hitting the cache as well:
```suggestion
    @quickcache(['domain_name'], timeout=60 * 60, skip_arg=lambda *args: settings.ENTERPRISE_MODE)
```

Reply:
makes sense